roles/commcare_analytics/templates/superset/superset_config.py.j2

```diff
@@ -75,7 +75,7 @@ RESULTS_BACKEND = RedisCache(
 
 
 class CeleryConfig(object):
-    accept_content = ['pickle']
+    accept_content = ['application/json']
```

Comment:
Should we support both pickle and application/json?

Reply:
We could. Not good with celery internals, I was wondering if the `pickle` woulld be needed for the  task in CCA ?

Reply:
Resolved in https://github.com/dimagi/commcare-analytics-ansible/pull/22/commits/94b4a0e16ed21b288aafcb93631de9800f2b1489

Reply:
Yeah, me neither, but thought maybe we shouldn't remove pickle altogether...might be wrong